packages/entities/entities-upstreams-targets/docs/upstreams-config-card.md

```diff
@@ -0,0 +1,112 @@
+- `@kong-ui-public/i18n` must be available as a `dependency` in the host application.
+- `axios` must be installed as a dependency in the host application
+
+## Usage
+
+### Install
+
+[See instructions for installing the `@kong-ui-public/entities-upstreams-tagrets` package.](../README.md#install)
+
+### Props
+
+#### `config`
+
+- type: `Object as PropType<KonnectUpstreamsEntityConfig | KongManagerUpstreamsEntityConfig>`
+- required: `true`
+- default: `undefined`
+- properties:
+    - `app`:
+        - type: `'konnect' | 'kongManager'`
+        - required: `true`
+        - default: `undefined`
+        - App name.
+
+    - `apiBaseUrl`:
+        - type: `string`
+        - required: `true`
+        - default: `undefined`
+        - Base URL for API requests.
+
+    - `requestHeaders`:
+        - type: `RawAxiosRequestHeaders | AxiosHeaders`
+        - required: `false`
+        - default: `undefined`
+        - Additional headers to send with all Axios requests.
+
+    - `workspace`:
+        - type: `string`
+        - required: `true`
+        - default: `undefined`
+        - *Specific to Kong Manager*. Name of the current workspace.
+
+    - `controlPlaneId`:
+        - type: `string`
+        - required: `true`
+        - default: `undefined`
+        - *Specific to Konnect*. Name of the current control plane.
+
+    - `entityId`:
+        - type: `string`
+        - required: `true`
+        - default: `''`
+        - The ID of the Upstream to display the config for
+
+The base konnect or kongManger config.
+
+#### `hideTitle`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Set this value to `true` to hide the card title.
+
+### Events
+
+#### fetch:error
+
+An `@fetch:error` event is emitted when the component fails to fetch the upstream. The event payload is the response error.
+
+#### fetch:success
+
+A `@fetch:success` event is emitted when the upstream is successfully fetched. The event payload is the Upstream object.
+
+#### loading
+
+A `@loading` event is emitted when loading state changes. The event payload is a boolean.
+
+#### copy:success
+
+A `@copy:success` event is emitted when a user clicks the `Copy JSON` button and the JSON object is copied successfully.
+
+### Usage example
+
+Please refer to the [sandbox](../sandbox/pages/UpstreamsConfigCardPage.vue). The page is accessible by clicking on the row or `View details` button of an existing Upstream.
+
+## TypeScript interfaces
+
+TypeScript interfaces [are available here](https://github.com/Kong/shared-ui-components/blob/main/packages/entities/entities-upstreams-targets/src/types/upstreams-config.ts) and can be directly imported into your host application. The following type interfaces are available for import:
```

Comment:
```suggestion
TypeScript interfaces [are available here](https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-upstreams-targets/src/types/upstreams-config.ts) and can be directly imported into your host application. The following type interfaces are available for import:
```

Could you search globally and replace `Kong/shared-ui-components` with `Kong/public-ui-components`? There're other `shared-ui-components` lying around.

Reply:
Links updated. Thanks!